srcpkgs/julia/template

```diff
@@ -2,66 +2,64 @@
 pkgname=julia
 version=1.5.0
 revision=1
-archs="i686* x86_64* armv7l aarch64"
+archs="i686* x86_64* armv7l* aarch64*"
```

Comment:
I didn't include musl for arm as [it's not a supported platform](https://julialang.org/downloads/#currently_supported_platforms), but if it works, it's fine.

Reply:
I see. None of the ARM stuff is tested at all because we don't have native ARM builders and, because the entire Julia build and dependency system is a total disaster, cross compilation fails. I can't see how `x86_64-musl` would build without patches but `{aarch64,armv7l}-musl` would fail. I'm tempted to leave it as is until somebody with a native ARM build finds a reason why the package is broken.

Reply:
Sounds good to me.